.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+on: [push]
+
+jobs:
+  test_job:
+    runs-on: ubuntu-latest
+    name: Tests the action
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Test Valid Configuration
+      uses: notrenderfarm/tflint-action@v1.2
+      with:
+        directory: 'valid_terraform'
+    # - name: Test Invalid Configuration
+    #   uses: notrenderfarm/tflint-action@v1.2
+    #   with:
+    #     directory: 'invalid_terraform'
```

Comment:
Should this be uncommented?

Reply:
Yes, the action will fail on the invalid configuration, causing the CI to fail.
I plan on running this test using a different test framework eventually, so that the CI as a whole is not affected by actions that should fail.